_posts/0000-01-12-Gymky.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Changing caption of my cool octocat
```

Comment:
Perhaps mention something Rainbow Dash would say? No idea what their catchphrase is 🤷‍♂